meetings/2021-07-28.md

```diff
@@ -0,0 +1,136 @@
+But each identity would have an associated inbox where they receive the notification about what they have access to.
+The authorization agent of Alice would have access to specific grants and not all applications would have access to all grants.
+
+Matthieu: I don't understand why the default from a security perspective is not 404.
+
+Sarven: First of all, 403 is defined and has meaning. If you can read the container and see that foo is linked to it, you already know that the resource exists. If you get that resource, you can get a 403, but since you know the resource exists 404 doesn't make sense. But the RFC also says you can use 404 to hide information.
+You're describing the purpose of hiding information.
+But you're capable of finding out on the request that a resource exists. So you're not hiding.
+
+Henry: Sorry it's boring me, we can't discuss some issues like that, it's wasting my time. Let's open an issue for it.
+
+Sarven: The first thing I said is whether the evaluation should take this into account.
+
+Henry: If you put up this question, we can put up a Use Case and have a user story where this is an issue and show how the different systems work. But out of the blue like that, we don't have anything that can be constructively discussed. We could make it so through a use case. It seems overly paranoid to me as a UC. We could put that in the Thor UC which doesn't allow information leakage through slash semantics. You're gonna have to put things behind Thor if you really want to be secure but it requires a lot of other things to be secure.
+
+Sarven: Just know that it comes up in the community group as a discussion we're trying to figure out. The more input we have, the better.
+Not only is it a security threat, but it's also valuable to answer.
+
+Henry: Yes, but we need priorities. Not everyone being on centralised systems is the first one. We can't start with security perfection.
+If I were to pay people to tink about decentralised networks, I wouldn't start with things that are almost unimportant for now, at this stage, I think it comes later. There are leaks and security issues everywhere.
+Are we writing software that is so well written that we're looking at marginal security use cases? Probably not yet.
+For me it's really important to have the Thor example (Solid behind Tor) and that's when the security people will come in and fix things.
+
+
+### UC3
+
+Elf: I think we should focus on inheritance.
+
+Matthieu: Let's merge as is and dedicate a whole conversation and proper Use Cases to required credentials discovery since it is a strong thread and you too Henry have pointed that the question has not been answered previously in similar experiments.
+
+
+### ACL Control
+
+Henry: In a way Control gives you access to not the resource but the control of that resource. Instead of having control, you could get read and write over an acl. That makes me think you could deprecate Control where the Read and Write.
```

Comment:
```suggestion
Henry: In a way, Control gives you access to, not the resource, but the control of that resource. Instead of having control, you could get read and write over an ACL. That makes me think you could deprecate Control where the Read and Write.
```